tech/languages/r/r-packages.md

```diff
@@ -0,0 +1,69 @@
+---
+title: R packages
+subsection: r
+order: 2
+---
+
+# Installing packages
+
+Recommended R packages are included as part of the 'R-core' component.
+A number of popular add-on packages from [CRAN](https://cran.r-project.org/), [Bioconductor](https://bioconductor.org/) and other sources are readily available via the official repositories.
+They are named with an `R-` prefix, such as `R-ggplot2`. Hence, running
+
+```bash
+$ dnf repoquery --repo=fedora-source R-*
```

Comment:
What about the following?

```bash
$ sudo dnf repoquery --repo=fedora-source 'R-*'
```

The main problem of the search is that this lists many many more things: R-core, R-java, R-devel, etc. (which are _not_ an R package), but also two versions of each compiled package (i686 and x86_64), plus R-*-devel subpackages... So that's _not_ a comprehensive list of R packages. The command above however _does_ list all R packages available in Fedora and nothing more.

Reply:
Yes, has a bit less accuracy (more churn), but has a much more informative character - the summaries - and is much more user-friendly (not a source-package).

Let me think about this. Maybe there's a middle ground (while avoiding a long-long command which would solve it).

Reply:
I've tried several things. The only command that comes close to showing the correct information (i.e., a list of R packages and nothing more, nothing duplicated) apart from repoquery is `sudo dnf list available --repo=fedora-source 'R-*'`, which _even longer_.

There are 368 R packages in Fedora. With the search command, I obtain 1132 results (some of them are even python packages!). Even if I add the `--repo=fedora-source` filter, the search command shows 1058 entries! The dnf command is great in general, but we have to admit that the search capability is not that good. At least when you know what you are looking for and you just want to obtain those packages. This is the case here.